tests/validators/sections/test_section_validator.py

```diff
@@ -1,4 +1,5 @@
 from app import error_messages
```

Comment:
From the card: "We will only ever have one list collector per list", we need to add such validation.

Reply:
Ah fair point, working on this now

Reply:
Validation for this now added, covers multiple list collectors referring to lists with repeating blocks, within the same section, and in different sections.